content/roadmap/now.md

```diff
@@ -0,0 +1,9 @@
+## Now
+
+Things we are working on currently:
+
+- Form component that details guidance around form design, validation and error text.
+- Full page form template example
+- Add more detailed documentation around existing templates
+- Automate documentation workflow
```

Comment:
@sukhrajghuman Maybe we should reword this to `Improve component architecture to allow for extendability, performance and automation` ? 

Maybe that can just be another point(s)